lib/fizzy/execute.cpp

```
@@ -154,11 +154,25 @@ Instance instantiate(const Module& module)
     constexpr unsigned pages = 1;
     // NOTE: fill it with zeroes
     bytes memory(pages * pagesize, 0);
-    return {module, std::move(memory)};
+    // TODO: add imported globals first
+    std::vector<uint64_t> globals;
+    globals.reserve(module.globalsec.size());
+    for (auto const& global : module.globalsec)
+    {
+        if (global.init_type == GlobalInitType ::constant)
+            globals.emplace_back(global.init.value);
+        else
+        {
+            // TODO: initialize by imported global
```

Can you throw a runtime exception here?